doc/dev_hello_world.md

```diff
@@ -139,21 +137,22 @@ metadata:
 
 ## Make the docker image
```

Comment:
Do we need to update to `Docker` in the title?

Reply:
Good point. I also referred to their [docs](https://docs.docker.com/glossary/), and it turns out both "Docker" and "docker" are acceptable. So for this one I just let it be. :)